CONTRIBUTING.md

```diff
@@ -4,23 +4,43 @@
 
 We use [pnpm](https://pnpm.io) to manage our dependencies.  
 You can [install it](https://pnpm.io/installation) using your preferred
-method, but we recommend using [Corepack](https://nodejs.org/api/corepack.html#supported-package-managers) so that you
-keep your package manager version in sync with ours.
+method, but we recommend using [Corepack](https://nodejs.org/api/corepack.html#supported-package-managers) so that you keep your package manager version in sync with ours.
 
 Make sure you run **Node 20** or the dependencies' installation will fail.  
 We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage your Node version.
```

Comment:
For windows's user : https://www.freecodecamp.org/news/nvm-for-windows-how-to-download-and-install-node-version-manager-in-windows-10/